pkg/client/preflight.go

```diff
@@ -62,6 +62,22 @@ func (c *SonobuoyClient) PreflightChecks(cfg *PreflightConfig) []error {
 		return []error{err}
 	}
 
+	preflightChecks := make(map[string]func(kubernetes.Interface, *PreflightConfig) error)
+	for key, value := range validPreflightChecks {
+		preflightChecks[key] = value
+	}
+
+	for _, name := range cfg.PreflightChecksSkip {
+		if strings.ToLower(name) == "false" {
```

Comment:
nit: You're using `strings.ToLower` 3 times here; I'd just create a new variable `lName` for the lower name to use in this narrow context.

I'd also say that if they provide `false` we should just run all the preflight checks. Its a weird situation since a user shouldn't ever do that `--skip-preflight=foo --skip-preflight=false,bar`. They're saying to skip foo and bar but not skip any?

Regardless, I'd rather that if they said `false` at all, we just allow all valid preflight checks.

You can add a `logrus.Trace(...)` statement there to expose that logic in the logging if you want. Its a good idea to surface little branches in logic like that.

Reply:
Added a loop that lowercases all preflight check names, and checks to see if a "false" check was passed in the slice which will cause all checks to execute.